content/stack/release-notes/redisbloom/redisbloom-2.4-release-notes.md

```diff
@@ -11,11 +11,52 @@ aliases: /modules/redisbloom/release-notes/redisbloom-2.4-release-notes/
 ---
 ## Requirements
 
-RedisBloom v2.4.5 requires:
+RedisBloom v2.4.8 requires:
 
 - Minimum Redis compatibility version (database): 6.0.16
 - Minimum Redis Enterprise Software version (cluster): 6.2.8
 
+## v2.4.8 (January 2024)
+
+This is a maintenance release for RedisBloom 2.4
+
+Update urgency: `HIGH`: There is a critical bug that may affect a subset of users. Upgrade!
+
+Details:
+
+- Bug fixes:
+
+  - [#727](https://github.com/RedisBloom/RedisBloom/pull/727) Potential crash on `CF.LOADCHUNK` (MOD-6344) - Additional fixes
```

Comment:
```suggestion
  - [#727](https://github.com/RedisBloom/RedisBloom/pull/727) Potential crash on `CF.LOADCHUNK` (MOD-6344) 
  - Additional fixes
```

Reply:
No; this bug was partially fixed in the previous version. these are additional fixes to the same bug

Reply:
Maybe this would help make it more clear?
```suggestion
  - [#727](https://github.com/RedisBloom/RedisBloom/pull/727) Potential crash on `CF.LOADCHUNK` (MOD-6344) and additional fixes
```

Reply:
Oh, wait. I think maybe I was still confused. So maybe:
```suggestion
  - [#727](https://github.com/RedisBloom/RedisBloom/pull/727) Additional fixes for potential crash on `CF.LOADCHUNK` (MOD-6344)
```

Reply:
In the previous version, we fixed this bug (727) but we then discovered that our fix was partial. In this version, we applied additional fixes to the same bug.